cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/etl/batch/BatchETLDBTest.java

```diff
@@ -345,29 +354,43 @@ public void testUserNamePasswordCombinations() throws Exception {
 
     Map<String, String> baseSourceProps = ImmutableMap.of(
       Properties.DB.CONNECTION_STRING, hsqlDBServer.getConnectionUrl(),
+      Properties.DB.TABLE_NAME, "my_table",
       Properties.DB.JDBC_PLUGIN_NAME, "hypersql",
       Properties.DB.IMPORT_QUERY, importQuery,
       Properties.DB.COUNT_QUERY, countQuery);
 
     Map<String, String> baseSinkProps = ImmutableMap.of(
       Properties.DB.CONNECTION_STRING, hsqlDBServer.getConnectionUrl(),
+      Properties.DB.TABLE_NAME, "my_table",
       Properties.DB.JDBC_PLUGIN_NAME, "hypersql",
-      Properties.DB.TABLE_NAME, "table",
-      Properties.DB.COLUMNS, "*");
+      Properties.DB.COLUMNS, "*",
```

Comment:
Good call. Currently, we allow users to pass in ``*`` (even that is not documented in the field description though). However, maybe we can make the property nullable and have ``*`` as the default value. Will make the change.